lib/fs.js

```diff
@@ -465,11 +486,16 @@ fs.readFileSync = function(path, options) {
 
 // Used by binding.open and friends
 function stringToFlags(flag) {
```

Comment:
Note: this function used to throw. Now it just returns undefined when there's an issue. Errors are thrown or returned to a callback by throwFlagsError.